src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -192,17 +212,48 @@ USER $_DEV_CONTAINERS_IMAGE_USER
 
 export function getFeatureLayers(featuresConfig: FeaturesConfig) {
 	let result = '';
-	const folders = (featuresConfig.featureSets || []).map(x => getSourceInfoString(x.sourceInformation));
+
+	// Features version 1  FIX
```

Comment:
This can cause issues since it breaks the ordering of how features are executed, adding features v1 first and then features v2.
There is no easy way to fix it since some feature sets might need to be called together at the same time and that already breaks the convention.

One option would be to execute v2 first and ensure that we include a cache of v2 at the same time but its still an issue for user written features.

Reply:
Features shouldn't depend on the order they are executed. Those that do we should take a look at.

Reply:
The only one so far is the Jupyter feature that requires some sort of Python installed.